### ISSUE TITLE
Booting with LayeredFS + Sys-netcheat locks up console at "Nintendo Switch" logo

### ISSUE THREAD

**Issue by Targren** [jakibaki/sys-netcheat#18]
Sys-Netcheat Version: 1.3.1
Console Firmware: 5.1.0 
Hekate version: 4.2
Atmosphere: 2018.09.18 (SDFiles release) 

Ini entry: 
`[LayeredFS + sys-netcheat]  
kip1=modules/newfirm/loader.kip  
kip1=modules/newfirm/sm.kip  
kip1=modules/newfirm/fs_mitm.kip  
kip1=modules/newfirm/pm.kip
kip1=modules/sys-netcheat.kip
secmon=.../exosphere.bin
atmosphere=1
kip1patch=nosigchk
debugmode=1
fullsvcperm=1`  

I tried both with and without the "secmon=" line, same result.

**Comment by jakibaki** [comment]
Fixed in 1.3.2! Thanks for your report!

**Comment by GiovanH** [comment]
I am experiencing this same issue again with 6.2.0

Environment:
Hekate 4.5

```
[Kosmos v11 w/ fptd]
kip1=modules/required/loader.kip
kip1=modules/required/pm.kip
kip1=modules/required/sm.kip
kip1=modules/required/fs_mitm.kip
kip1=modules/sysftpd/sys-ftpd.kip
secmon=modules/required/exosphere.bin
kip1patch=nosigchk
atmosphere=1

[Kosmos v11 sys-netcheat on 5555]
kip1=modules/required/loader.kip
kip1=modules/required/pm.kip
kip1=modules/required/sm.kip
kip1=modules/required/fs_mitm.kip
kip1=modules/sysnetcheat/sys-netcheat.kip
secmon=modules/required/exosphere.bin
kip1patch=nosigchk
debugmode=1
atmosphere=1
```
The first entry works, while the second doesn't. The second either hangs on the switch logo or crashes with error code 2003-0008
